website/blog/2024-01-16-running-tabby-locally-with-rocm.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: running-tabby-locally-with-rocm.md
+title: Running Tabby Locally with AMD ROCm
+authors:
+  - name: Julien Marcuse
+    url: https://github.com/boxbeam
+    image_url: https://github.com/boxbeam.png
+tags: [deployment]
+---
+
+For those using (compatible) AMD graphics cards, you can now run Tabby locally with GPU acceleration using AMD's ROCm toolkit.
+
+Currently, Tabby with ROCm is only supported on Linux, and can only be run directly from a compiled binary. In the future, Tabby will be able to run with ROCm on Windows, and we will distribute a Docker container capable of running with ROCm on any platform.
+
+Before starting, please make sure you are on a supported system and have ROCm installed. The AMD website [details how to install it](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/install-overview.html), find the instructions for your given platform.
```

Comment:
Provide examples / screenshots for your local installation of rocm

Reply:
Wasn't really sure what to provide, so I just showed the necessary rocm packages installed on my system.